### ISSUE TITLE
Get-CCPCredential Parameter set cannot be resolved

### ISSUE THREAD

**Issue by arkayali** [pspete/CredentialRetriever#33]
## Your Environment

* PowerShell Version: 5
* CredentialRetriever Branch & Version: 3.9.48
* CyberArk Version:11.5

## Expected Behaviour

To be able to fetch the password using user / cert Auth 
It works fine with user auth only 

## Current Behaviour
Fetching the password works fine without adding Cert Auth.
When adding Cert Auth and adding -CertificateThumbPrint  to the cmdlet , I get an error : Parameter set cannot be resolved
Removing -CertificateThumbPrint would result with an error : Failed to verify application 

## Sample Output
$apiCred = Get-CCPCredential -AppID $AppID -safe $safe -URL $CCP_s -username $user -WebServiceName $AIM -UseDefaultCredentials -CertificateThumbPrint $CertThumb 

![image](https://user-images.githubusercontent.com/44178884/190689998-52f79767-68d4-48ed-be3c-7ea3577a47a9.png)

$apiCred = Get-CCPCredential -AppID $AppID -safe $safe -URL $CCP_s -username $user -WebServiceName $AIM -UseDefaultCredentials 

![image](https://user-images.githubusercontent.com/44178884/190690025-3c070cbe-f6c9-4e55-be7d-331c41f12de6.png)

**Comment by pspete** [comment]
Here is the equivalent command in using `Invoke-RestMethod`:
```powershell
Invoke-RestMethod -Uri "$CCP_s/$AIM/api/Accounts?AppID=$AppID&Safe=$safe&UserName=$user" -UseDefaultCredentials -CertificateThumbPrint $CertThumb
```

**Comment by pspete** [comment]
ParameterSet restrictions now removed - 
Changes available in version `3.10.56`.
Closing
